### DISCUSSION TITLE
Apple App Clips Experience?

### DISCUSSION THREAD

**Discussion by cselin** [MobileNativeFoundation/discussions#126] [General]
I'm curious if anyone would care to share their experience in integrating Apple App Clips and how they have performed?

**Comment by devandanger** [comment]
I haven't made updates to the apps I support at this point for App Clips.  My experience has been just looking into the steps I need to take in order to accomplish it at this point.  

These steps are mostly organization / modularization considerations.  Apps with a heavily dependency graph, I've had to reconsider the value of those libraries and even the mechanism (Cocoapods/SPM).  I've found Cocoapods to be great, but feel SPM might be a simpler alternative to manage that.

So again lot more planning oppose to actually doing it at this point.
